### DISCUSSION TITLE
Add a Field type for selecting media folders

### DISCUSSION THREAD

**Discussion by sander1095** [PiranhaCMS/piranha.core#1710] [Ideas]
Currently I have written a custom data select field that gets all the media folders recursively and the amount of items in them, for example: `/images/employees/ (9)`, `/images/ (0)`, `/ (0)`

This is useful because I want to let the cms user select a folder whose media should be displayed, perhaps randomly. Currently I need to either select the media myself, which makes randomness impossible, and change my selection if I ever add an image. 

Selecting a folder and saving the ID would be super useful!

I can post my current solution with the data select field item if anyone is interested!

**Comment by tidyui** [comment]
Moving this to discussions
